GooglePlayInstant/Samples/SphereBlast/scripts/BallMovement.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class BallMovement : MonoBehaviour {
+    private float _speed = 20F;
+    private float _boostMultiplier = 100F;
+    private float _keyDownMultiplier = 10F;
+    private Rigidbody _rb;
+    private BaseGame _baseGame;
+    private string[] _outOfBoundsObjectNames = new string[] {
+        "OOBNorth",
+        "OOBEast",
+        "OOBSouth",
+        "OOBWest",
+        "OOBBottom",
+        "OOBTop"
+    };
+
+    void Start () {
+        _rb = GetComponent<Rigidbody> ();
+        _baseGame = GameObject.Find ("BaseGame").GetComponent<BaseGame> ();
+    }
+
+    void FixedUpdate () {
+        Vector3 acc = Input.acceleration;
```

Comment:
Prefer var here and for local variables below

Reply:
fixed

Reply:
Just in case it takes longer to merge Quick Deploy